course_catalog/api.py

```diff
@@ -565,6 +735,36 @@ def sync_ocw_courses(
             log.exception("Error encountered parsing OCW json for %s", course_prefix)
 
 
+def sync_ocw_next_courses(*, course_prefixes, force_overwrite, start_timestamp=None):
```

Comment:
I think we can force this into the ETL pipeline, although we would need to add an extra step to the pipeline in order to load the course resources at the same time as we load the course instead of as a separate task (the way we do for xpro). I think rewriting this code to fit the ETL pipeline pattern will be a lot of work though, and since the only live resource that uses that pattern is podcasts, I don't think that work is worth it.